src/targets/browser/index.jsx

```diff
@@ -155,6 +157,18 @@ export const initApp = () => {
       cozyClient: client,
       isPublic: isPublic
     })
+    Sentry.init({
+      dsn: 'https://16c26a60d9019eea9d9a9775573e3765@errors.cozycloud.cc/73',
+      environment: process.env.NODE_ENV,
+      release: manifest.version,
+      integrations: [
+        new CaptureConsole({ levels: ['error'] }) // We also want to capture the `console.error` to, among other things, report the logs present in the `try/catch`
```

Comment:
Sry I read too fast the first time but if you want to use `withSentryReactRouterV6Routing` you need also to enable `reactRouterV6BrowserTracingIntegration` : 

imports : 
```
import {
  useLocation,
  useNavigationType,
  createRoutesFromChildren,
  matchRoutes
} from 'react-router-dom'
```

code for sentry init :
```
integrations: [
    Sentry.reactRouterV6BrowserTracingIntegration({
      useEffect: React.useEffect,
      useLocation,
      useNavigationType,
      createRoutesFromChildren,
      matchRoutes,
    }),
...
],
```

if you like more explanation there is this documentation : https://docs.sentry.io/platforms/javascript/guides/react/features/react-router/

Reply:
@cballevre updated